src/styles/getMuiTheme.js

```diff
@@ -109,6 +109,16 @@ export default function getMuiTheme(baseTheme, muiTheme) {
     },
     paper: {
       backgroundColor: palette.canvasColor,
+      zDepthShadows: [null, ...[
```

Comment:
What about remove the `null` and the spread operator. I think that it would be better to use
```
boxShadow: this.state.muiTheme.paper.zDepthShadows[zDepth - 1],
```